.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+Reviewers: GitHub @<name>s of all reviewers
+
+Linked PRs: <links to corresponding PRs, optional>
+
+## Changes
+- <changes here>
+
+## TODOs:
+- [ ] +1
+- [ ] Updated README
+- [ ] Updated CHANGELOG
+- [ ] <Other applicable TODOs>
+- [ ] <Feedback from reviewers>
+
+## How To Test
+- <necessary config changes>
+- <necessary corresponding PRs>
+- <how to access the new / changed functionality -- fixtures, URLs>
+
+## Applicable Research Resources
+- <links, optional>
```

Comment:
Not a bad idea @MikeKlemarewski . My only concerns are:
- When a PR template should have steps specific to its individual plugin/repo that a more general PR template in code-style doesn't address
- Would github's auto-populating of PRs with the template work if it's just linked to another repo? IMO the main benefit of the templates are their auto-populating nature.